.github/workflows/ci.yml

```diff
@@ -282,24 +395,32 @@ jobs:
   # a tag is pushed.
   publish-release:
     if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags/v') && !github.event.repository.fork
-    # Depends explictly on the 'package' job so we can access its output
-    needs: [all-required-checks-done, package]
+    # Depends explictly on the 'check-version' job so we can access its output
+    needs:
+      - all-required-checks-done
+      - check-version
     runs-on: ubuntu-latest
     steps:
       - name: Download NuGet package artifact
         uses: actions/download-artifact@v2
         with:
           name: nuget-packages
+      - name: Download Samples artifact
+        uses: actions/download-artifact@v2
+        with:
+          name: samples
 
       # if tag contains "-" treat it as pre-release
       # example: 1.0.0-beta1
       - name: Create release
         uses: softprops/action-gh-release@v1
         with:
-          name: Release v${{ needs.package.outputs.version }}
+          name: Release v${{ needs.check-version.outputs.version }}
           draft: true
-          prerelease: ${{ contains(needs.package.outputs.version, '-') }}
-          files: "*.nupkg"
+          prerelease: ${{ contains(needs.check-version.outputs.version, '-') }}
+          files: |
+            "*.nupkg"
+            "Samples-*.zip"
```

Comment:
These glob patterns do not work unfortunately:
![image](https://user-images.githubusercontent.com/3692455/129563289-1f19cae2-1a88-44a6-a757-a18635adc12f.png)

Here is a fix I tested on my fork:
```suggestion
            *.nupkg
            Samples-*.zip
```

Reply:
Thanks @jgiannuzzi. That's now fixed.